src/app/hearing-part/effects/listing-create.effects.ts

```diff
@@ -18,7 +19,10 @@ export class ListingCreateEffects {
         ofType<CreateListingRequest>(HearingPartActionTypes.CreateListingRequest),
         mergeMap(action =>
             this.hearingPartService.createListing(action.payload).pipe(
-                mergeMap(() => [new CreateComplete(), new Notify(LISTING_REQUEST_CREATED)]),
+                concatMap(() => [
+                    new fromNotes.CreateMany(action.payload.notes),
```

Comment:
are you sure we gonna receive notes in payload? aren't they supposed to go with a separate call?

Reply:
action.payload is not returned from this.hearingPartService.createListing function. It is entire listing request to be created, so action.payload.notes are the notes defined in frontend and they are dispatched as CreateMany action. CreateMany is handled in NotesEffects